Tests/NetworkServiceTests.swift

```diff
@@ -122,6 +122,18 @@ class NetworkServiceTests: XCTestCase {
         waitForExpectations(timeout: 1.0, handler: nil)
     }
     
+    func test_NetworkService_ConvenienceInit() {
+        
+        let _ = NetworkService(session: URLSession.shared, networkActivityIndicatable: MockNetworkActivityIndicator())
+        
+        XCTAssert(true)
+    }
+    
+    func test_NetworkServiceHelper_InvalidHTTPResponsErrorUnknownError() {
+        let _ = NetworkServiceHelper.networkServiceFailure(for: NSError(domain: NSURLErrorDomain, code: NSURLErrorBadURL, userInfo: nil))
+        
+        XCTAssert(true)
```

Comment:
same here, should probably test that the object is not nil

Reply:
adding a check for the error